storey/table.py

```diff
@@ -1540,6 +1585,12 @@ def __init__(self):
 class _PersistJob:
     def __init__(self, key, data, callback, extra_data=None):
         self.key = key
-        self.data = data
+        self.data = copy.deepcopy(data)
```

Comment:
Looks like this is used only from NoSqlTarget, when periodic flushing is disabled. So why do we need to deepcopy it?

Reply:
I don't want a mixture of copied/not copied members. so I deep copy all